src/helix/endpoints/ccls/get_content_classification_labels.rs

```diff
@@ -0,0 +1,187 @@
+    #[cfg(feature = "twitch_oauth2")]
+    const SCOPE: twitch_oauth2::Validator = twitch_oauth2::validator![];
+}
+
+impl RequestGet for GetContentClassificationLabelsRequest<'_> {}
+
+#[cfg(test)]
+#[test]
+fn test_request() {
+    use helix::*;
+    let req = GetContentClassificationLabelsRequest::new();
+
+    // From api call
+    let data = br#"
+    {
+      "data": [
+        {
+          "description": "Discussions or debates about politics or sensitive social issues such as elections, civic integrity, military conflict, and civil rights in a polarizing manner.",
+          "id": "DebatedSocialIssuesAndPolitics",
+          "name": "Politics and Sensitive Social Issues"
+        },
+        {
+          "description": "Excessive tobacco glorification or promotion, any marijuana consumption/use, legal drug and alcohol induced intoxication, discussions of illegal drugs.",
+          "id": "DrugsIntoxication",
+          "name": "Drugs, Intoxication, or Excessive Tobacco Use"
+        },
+        {
+          "description": "Participating in online or in-person gambling, poker or fantasy sports, that involve the exchange of real money.",
+          "id": "Gambling",
+          "name": "Gambling"
+        },
+        {
+          "description": "Games that are rated Mature or less suitable for a younger audience.",
+          "id": "MatureGame",
+          "name": "Mature-rated game"
+        },
+        {
+          "description": "Prolonged, and repeated use of obscenities, profanities, and vulgarities, especially as a regular part of speech.",
+          "id": "ProfanityVulgarity",
+          "name": "Significant Profanity or Vulgarity"
+        },
+        {
+          "description": "Content that focuses on sexualized physical attributes and activities, sexual topics, or experiences.",
+          "id": "SexualThemes",
+          "name": "Sexual Themes"
+        },
+        {
+          "description": "Simulations and/or depictions of realistic violence, gore, extreme injury, or death.",
+          "id": "ViolentGraphic",
+          "name": "Violent and Graphic Depictions"
+        }
+      ]
+    }
+    "#
+    .to_vec();
+
+    let http_response = http::Response::builder().body(data).unwrap();
+
+    let uri = req.get_uri().unwrap();
+    assert_eq!(
+        uri.to_string(),
+        "https://api.twitch.tv/helix/content_classification_labels?"
+    );
+
+    let res = GetContentClassificationLabelsRequest::parse_response(Some(req), &uri, http_response)
+        .unwrap()
+        .data;
+
+    assert_eq!(res.len(), 7);
+
+    assert_eq!(res[0].description, "Discussions or debates about politics or sensitive social issues such as elections, civic integrity, military conflict, and civil rights in a polarizing manner.");
+    assert_eq!(res[0].name, "Politics and Sensitive Social Issues");
+
+    assert_eq!(
+        res[0].id,
+        types::ContentClassificationId::Other("DebatedSocialIssuesAndPolitics".to_string())
+    );
```

Comment:
Not worth a twitch-types release imo

Reply:
it can be added easily without a break because it's non_exhaustive